hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1027,6 +1027,7 @@ message OpenKeyBucket {
 message OpenKey {
   required string name = 1;
   required uint64 clientID = 2;
+  optional uint64 parentID = 3;
```

Comment:
Lets make the proto changes in #3226 and merge that first, then update this PR with the new proto.